grazer/src/main/java/edu/asu/diging/grazer/core/fileupload/db/impl/FileUploadDatabaseConnection.java

```diff
@@ -0,0 +1,53 @@
+package edu.asu.diging.grazer.core.fileupload.db.impl;
+
+import java.util.List;
+import org.hibernate.SessionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import edu.asu.diging.grazer.core.domain.impl.FileTransformationImpl;
+import edu.asu.diging.grazer.core.fileupload.db.IFileUploadDatabaseConnection;
+
+@Component
+@Transactional
+public class FileUploadDatabaseConnection implements IFileUploadDatabaseConnection {
```

Comment:
what is the database connection storing? there is no FileUpload class, is there?

Reply:
It stores the uploader, label, description and date of the files uploaded